MeetU/MeetU/Content/Spinner.less

```diff
@@ -3,8 +3,8 @@
 .cssload-loader {
     margin: 30px auto;
 	position: relative;
-	width: 62px;
-	height: 62px;
+	width: 31px;
```

Comment:
you should define another class. when you add some new feature, you shouldnt change current behaviours